addon/components/positioned-container.js

```diff
@@ -31,7 +31,9 @@ export default Ember.Component.extend({
   updateAlignment: function() {
     var alignmentTarget = this.get('alignmentTarget');
     if (Ember.typeOf(alignmentTarget) === 'string') {
+      let alignmentTargetSelector = alignmentTarget;
       alignmentTarget = Ember.$(alignmentTarget)[0];
```

Comment:
Can we use `alignmentTargetSelector` on the right hand side of this line to make this function more readable?